src/main/java/com/coreos/jetcd/data/Header.java

```diff
@@ -9,15 +9,12 @@
   private final long memberId;
   private final long revision;
   private final long raftTerm;
-  private final long compactRevision;
```

Comment:
header is tied to watch impl. this change will break ci. it must be changed along with watch impl.